### PR TITLE
MicroHacks for Managed Kubernetes Service (AKS) and Microservices with Azure Container Apps

### PR DESCRIPTION
**Pull request by judithmh**

### CODE REVIEW

**Comment by judithmh** [comment]
@microsoft-github-policy-service agree company="MaibornWolff GmbH"

**Review by rajata07 — CHANGES_REQUESTED** [review]
The links for Solution - Spoilerwarning for Challenges does not work. I think you miss a dot in the beginning in every file.

File: 03-Azure/01-01-App Innovation/02_OperationalizeAKSCluster/Challenges/02-Deploy-and-configure.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Success Criteria
+
+- Config for AKS merged in local kubeconfig
+- BusyBox container running and viewable via kubectl
+- Container was accessed via kubectl
+
+## Learning resources
+
+- [Connect to AKS via CLI](https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli)
```
Comment (rajata07) [inline 03-Azure/01-01-App Innovation/02_OperationalizeAKSCluster/Challenges/02-Deploy-and-configure.md:23]: How about replacing the link with following:
https://learn.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-cli#connect-to-the-cluster

File: 03-Azure/01-01-App Innovation/02_OperationalizeAKSCluster/Challenges/01-Setup-Environment.md [hunk 2]
```diff
@@ -29,4 +29,4 @@ In this challenge we will setup the basic Azure resources we need to fulfill our
 
 ## Solution - Spoilerwarning
 
-Challenge 1: [Setup the environment](./Solutionguide/01-Setup-Environment-solution.md)
+Challenge 1: [Setup the environment](../Solutionguide/01-Setup-Environment-solution.md)
```
Comment (rajata07) [inline 03-Azure/01-01-App Innovation/02_OperationalizeAKSCluster/Challenges/01-Setup-Environment.md:32]: Sorry, I just noticed that links to next Challenge under Solution guides also do not work. A dot is also missing there. For example:

## Next Challenge

Challenge 2: [Deploy and configure your first pods](./Challenges/02-Deploy-and-configure.md)

**Comment by cthoenes** [comment]
@judithmh can you check for the last comment pls? :)

File: 03-Azure/01-01-App Innovation/02_OperationalizeAKSCluster/Challenges/01-Setup-Environment.md (hunk 2 above)
Comment (judithmh) [reply 03-Azure/01-01-App Innovation/02_OperationalizeAKSCluster/Challenges/01-Setup-Environment.md:32]: I am sorry for the delay - I just updated the links accordingly

**Review by rajata07 — APPROVED** [review]

**Review by rajata07 — APPROVED** [review]

**Review by rajata07 — APPROVED** [review]

**Review by rajata07 — APPROVED** [review]

**Comment by cthoenes** [comment]
Thank you very much @judithmh and @rajata07

**Pull request merged**